package.json

```diff
@@ -13,18 +13,24 @@
   "scripts": {
     "docs": "npm run build && node lib/bin/docs.js",
     "preversion": "git pull && npm test",
-    "version": "npm run docs && npm run gen-completions && git add docs/* goodies/* README.md",
+    "version":
+      "npm run docs && npm run gen-completions && git add docs/* goodies/* README.md",
```

Comment:
Not sure if this is as attended, but on commit it started formatting these files.

Reply:
mostly intended. maybe should exclude this, since it gets reformatted when it's modified by npm